olm_deploy/operatorregistry/elasticsearch-operator.package.yaml

```diff
@@ -1,9 +1,9 @@
 packageName: elasticsearch-operator
 defaultChannel: "stable"
 channels:
-- name: "5.2"
-  currentCSV: elasticsearch-operator.v5.2.0
+- name: "5.3"
+  currentCSV: elasticsearch-operator.v5.3.0
 - name: "stable"
-  currentCSV: elasticsearch-operator.v5.2.0
+  currentCSV: elasticsearch-operator.v5.3.0
```

Comment:
I don't recall if this `stable-5.2` needs to be modified too.

Reply:
Technically you are right, it should be `stable-5.3`. However this the package.yaml is used only for olm-deploy. Production deployments are fine as long as the annotations.yaml says:
```
  operators.operatorframework.io.bundle.channels.v1: stable,stable-5.3
```